source/features/hide-useless-comments.tsx

```diff
@@ -55,11 +57,112 @@ function unhide(event: React.MouseEvent<HTMLButtonElement>): void {
 	event.currentTarget.parentElement!.remove();
 }
 
+async function getMutedUsers(): Promise<string[]> {
+	return ((await optionsStorage.getAll() as Options).mutedUsers).split(' ');
+}
+
+async function setMutedUsers(mutedUsers: string[]): Promise<void> {
+	return optionsStorage.set({mutedUsers: mutedUsers.join(' ')});
+}
+
+function minimizeComment(comment: HTMLElement): void {
+	if (select.exists('.js-targetable-comment[id^="issue-"]', comment)) {
+		return;
+	}
+
+	select('.minimized-comment', comment)!.classList.remove('d-none');
+	select('.minimized-comment .Details-element', comment)!.removeAttribute('open');
+	select('.unminimized-comment', comment)!.classList.add('d-none');
+}
+
+function unminimizeComment(comment: HTMLElement): void {
+	if (select.exists('.js-targetable-comment[id^="issue-"]', comment)) {
+		return;
+	}
+
+	select('.minimized-comment', comment)!.classList.add('d-none');
+	select('.minimized-comment .Details-element', comment)!.removeAttribute('open');
```

Comment:
This line is identical in `minimizeComment`

Also is it possible to deduplicate all minimize/unminimize code? e.g. “toggleComment(comment, boolean)”

Reply:
> Also is it possible to deduplicate all minimize/unminimize code? e.g. “toggleComment(comment, boolean)”

We explicitly require `minimizeComment` at feature's initial run.

Reply:
?

`minimizeComment(comment)` would become `toggleComment(comment, true)`